packages/components/src/entities/SamplesEditButton.tsx

```diff
@@ -56,7 +56,7 @@ export const SamplesEditButton: FC<OwnProps & SampleGridButtonProps & RequiresMo
         async (onClick: () => void, errorMsg?: string): Promise<void> => {
             if (model?.hasSelections) {
                 setCrossFolderSelectionResult(undefined);
-                const result = await getCrossFolderSelectionResult(model.id, 'sample');
+                const result = await getCrossFolderSelectionResult(model.id, 'sample', model.filterArray.length > 0);
```

Comment:
Rather than `model.filterArray.length > 0` should this be using `model.isFiltered()`? The check is different:

```ts
get isFiltered(): boolean {
    return (
        this.baseFilters?.length > 0 ||
        this.filterArray?.length > 0 ||
        this.queryInfo?.getFilters(this.schemaQuery.viewName)?.size > 0
    );
}
```

If you do this there are a number of other places in these changes you may want to do the same thing.

Reply:
No, this is meant to capture when there are user-defined filters, not if there is any filtering.